src/main/java/controllers/PostControl.java

```diff
@@ -0,0 +1,98 @@
+package controllers;
+
+import com.scorpiac.javarant.Rant;
+import javafx.beans.property.StringProperty;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Created by Tahnik Mustasin on 19/02/2017.
+ */
+public class PostControl extends BorderPane{
```

Comment:
https://docs.oracle.com/javase/8/javafx/fxml-tutorial/custom_control.htm

I followed this to make it a reuseable control